src/components/DropZone/components/FileUpload/FileUpload.scss

```diff
@@ -7,6 +7,9 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
 .FileUpload {
   padding: $fileupload-padding;
   text-align: center;
+  &.isMeasuringSize {
+    display: none;
```

Comment:
ah, I thought it would get the correct measurement, as it's using the Dropzone node to get the width? because visibility hidden still takes up up space, I think you'd still see a jump from extra-large dimensions -> final dimensions in terms of the height 

![resizeInit](https://user-images.githubusercontent.com/3307989/68717441-cfde0280-0574-11ea-99ab-95ae3ad6d9d1.gif)

Reply:
:wave: Hi @laranoell, this was causing an issue because in a flex layout the with ended up begin 0, so since it wasn't released yet, I am removing it to get a release out. Just this hidden rule. If it does cause you issues, let me know.